release/models/mpls/openconfig-mpls-ldp.yang

```diff
@@ -259,6 +281,8 @@ module openconfig-mpls-ldp {
             "Neighbor state attributes.";
           uses mpls-ldp-neighbor-config;
           uses mpls-ldp-neighbor-state;
+          uses mpls-ldp-label-advertisement-config;
```

Comment:
In the config container...

Reply:
ACK - but I was wondering whether we can just have these leaves be part of the existing `mpls-ldp-neighbor-config` grouping rather than creating a new one.